proto/viam/app/data/v1/data.proto

```diff
@@ -420,6 +423,19 @@ message RemoveBoundingBoxFromImageByIDRequest {
 
 message RemoveBoundingBoxFromImageByIDResponse {}
 
+// UpdateBoundingBoxRequest updates the bounding box with specified bbox ID for the file represented by the binary id.
```

Comment:
I would just spell out bounding box ID instead of bbox ID and say binary data ID instead of binary id?